### DISCUSSION TITLE
Wow. 1000 OPEN bug reports.

### DISCUSSION THREAD

**Discussion by johnkw** [amzn/selling-partner-api-models#625] [Ideas]
It keeps getting harder to find things in there.  Oh well.  Just saying, would be nice to fix a lot of those.

**Comment by stefnats** [comment]
To be fair, lots of issues are still open but solved. 
Imagine all issues would get closed instantly, even if they won't be solved.

I'm not in any way affiliated with Amazon, those are just a few thoughts from a fellow dev.

**Comment by johnkw** [comment]
Yes, there are certainly many. Bug #133 is an example of a bug that should be closed because it's not longer relevant as there is a new endpoint version of that API.
